### ISSUE TITLE
Rviz2 Generating a Map Issue

### ISSUE THREAD

**Issue by tharukakk1** [turtlebot/turtlebot4#456]
### Robot Model

Turtlebot4 Standard

### ROS distro

Humble

### Networking Configuration

Simple Discovery

### OS

Ubuntu 22.04

### Built from source or installed?

Select One

### Package version

n/a

### Type of issue

LIDAR

### Expected behaviour

Rviz2 map should begin to generate and Map Display should not have warning status.

### Actual behaviour

Rviz2 does not show what the LiDAR/camera should be scanning. In Displays menu, Map has yellow exclamation with Warn Status "No map received". Global Status allow has error "Frame [map] does not exist". See attached image 
![IMG_3985](https://github.com/user-attachments/assets/3ee22305-7281-4f1f-a5b1-1d2de49b5308)

### Error messages

```bash
Map Status Warn: "No map received"
Global Status Error: "Frame [map] does not exist"
```

### To Reproduce

1. confirm turtlebot is running and all hardware is connected
2. confirm ros2 topic list is full $ ros2 topic list
3. $ source /opt/ros/humble/setup.bash
4. $ ros2 launch turtlebot4_navigation slam.launch.py
5. $ ros2 launch turtlebot4_viz view_robot.launch.py

### Other notes

The network does not have internet access due to university restrictions.
*Edit: rebooted host computer and robot to abate "Message Filter dropping message: frame 'rplidar_link' at time 1722986815.188 for reason 'discarding message because the queue is full'" messages.

**Comment by smatarCPR** [comment]
Hello @tharukakk1 ,
Thank you for reaching out on the Turtlebot 4 Github page. I am very sorry to hear that you are having issues with one of our products.

In terms of mapping:
1. When you list the topics using ros2 topic list: are the topics on the Turtlebot 4 and your computer match? Are you able to echo any topic and do you get the data? In particular, are you able to echo Lidar data?
2. Would it be possible to launch RVIZ (without mapping) and see if you can visualise the Lidar topic? 
3. Are you launching SLAM on the Turtlebot 4 or on your remote computer? Have you installed `sudo apt install ros-humble-turtlebot4-navigation` on your computer? What is the terminal output when launching navigation slam.launch.py?
4. For the error message you have noted, this could be an indicator that the times on the Turtlebot and your remote computer are not synched. We have the following recommendations with regards to checking timesync:

Setup NTP config Create3 to time sync Create3 and RPi - see instructions in https://github.com/turtlebot/turtlebot4/issues/216.
Setup NTP server on your remote PC on the network - see instructions [here](https://ubuntu.com/server/docs/use-timedatectl-and-timesyncd)
Set up the RPi to sync to that PC
Run the command: systemctl status systemd-timesyncd and see when the timesync last occurred.
For the Raspberry Pi, you can use systemctl status chrony to see when the last sync was and you can force a sync by restarting the chrony service.

Let me know how these recommendations work for you. Looking forward to hearing from you soon.
Best Regards,
Saif

**Comment by tharukakk1** [comment]
Hi @smatarCPR, sorry for the late reply, I came to the lab on W/R/F this week instead of M/T. I started up the robot and everything worked on the first try. It seems like all of these errors occurred because I was doing other work on the robot last week and didn't shut it down before starting RViz2. I've tried it multiple times now and it works perfectly. It still returns the same time-keeping error but Nav2 and mapping in RViz2 work anyways.
